bigml/__init__.py

```diff
@@ -1 +1,4 @@
-__path__ = __import__('pkgutil').extend_path(__path__, __name__)
+try:
+    __import__('pkg_resources').declare_namespace(__name__)
+except ImportError:
+    __path__ = __import__('pkgutil').extend_path(__path__, __name__)
```

Comment:
According to this docs (https://packaging.python.org/guides/packaging-namespace-packages/#pkgutil-style-namespace-packages) [See the note at the end of the page] it seems that this is not needed, and it is not advisable. 

`The idea behind this was that in the rare case that setuptools isn’t available packages would fall-back to the pkgutil-style packages. This isn’t advisable because pkgutil and pkg_resources-style namespace packages are not cross-compatible. If the presence of setuptools is a concern then the package should just explicitly depend on setuptools via install_requires.`

Reply:
I tried to remove it, and I could install both libraries without any problem with Python2 and Python3. It will only be a problem if setuptools is not installed. It would be a very uncommon case...